apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -3,43 +3,132 @@ defmodule Aecore.Structures.SpendTx do
   Aecore structure of a transaction data.
   """
 
+  @behaviour Aecore.Structures.Transaction
   alias Aecore.Structures.SpendTx
+  alias Aeutil.Serialization
+  alias Aeutil.Parser
+  alias Aecore.Structures.Account
+  alias Aecore.Chain.ChainState
+  alias Aecore.Wallet
 
+  require Logger
+
+  @typedoc "Expected structure for the Spend Transaction"
+  @type payload :: %{
+    to_acc: Wallet.pubkey(),
+    value: non_neg_integer(),
+    lock_time_block: non_neg_integer()
+  }
+
+  @typedoc "Reason for the error"
+  @type reason :: String.t()
+
+  @typedoc "Structure that holds specific transaction info in the chainstate.
+  In the case of SpendTx we don't have a subdomain chainstate."
+  @type tx_type_state() :: %{}
```

Comment:
Is this always empty? otherwise make specific

Reply:
In the case of SpendTx, yes it is always empty

Reply:
okay, good!